cdap-ui/app/cdap/components/PluginJSONCreator/Create/Content/Outputs/index.tsx

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright © 2020 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import { WithStyles } from '@material-ui/core/styles/withStyles';
+import { styles } from 'components/AbstractWidget/RadioGroupWidget';
+import Heading, { HeadingTypes } from 'components/Heading';
+import JsonMenu from 'components/PluginJSONCreator/Create/Content/JsonMenu';
+import PluginInput from 'components/PluginJSONCreator/Create/Content/PluginInput';
+import StepButtons from 'components/PluginJSONCreator/Create/Content/StepButtons';
+import {
+  CreateContext,
+  createContextConnect,
+  ICreateContext,
+} from 'components/PluginJSONCreator/CreateContextConnect';
+import * as React from 'react';
+
+const OutputsView: React.FC<ICreateContext & WithStyles<typeof styles>> = ({
+  classes,
+  pluginName,
+  pluginType,
+  displayName,
+  emitAlerts,
+  emitErrors,
+  configurationGroups,
+  groupToInfo,
+  groupToWidgets,
+  widgetToInfo,
+  widgetToAttributes,
+  jsonView,
+  setJsonView,
+  outputName,
+  setOutputName,
+}) => {
+  const [localOutputName, setLocalOutputName] = React.useState(outputName);
+
+  function saveAllResults() {
+    setOutputName(localOutputName);
+  }
+
+  return (
+    <div>
+      <JsonMenu
+        pluginName={pluginName}
+        pluginType={pluginType}
+        displayName={displayName}
+        emitAlerts={emitAlerts}
+        emitErrors={emitErrors}
+        configurationGroups={configurationGroups}
+        groupToInfo={groupToInfo}
+        groupToWidgets={groupToWidgets}
+        widgetToInfo={widgetToInfo}
+        widgetToAttributes={widgetToAttributes}
+        jsonView={jsonView}
+        setJsonView={setJsonView}
+        outputName={localOutputName}
+      />
+      <Heading type={HeadingTypes.h3} label="Outputs" />
+      <br />
+      <PluginInput
```

Comment:
It is a little misleading that we call the page as "Outputs" but allow the user to specify only one plugin property. Can we change the title to `Output` and prepopulate a plugin property that the user has already configured? They can configure a plugin property to have schema editor